partiql-lang/src/main/kotlin/org/partiql/lang/eval/ExprFunction.kt

```diff
@@ -98,9 +73,4 @@ interface ExprFunction {
  * @param session The current [EvaluationSession].
  * @param args The argument list.
  */
-fun ExprFunction.call(session: EvaluationSession, args: Arguments): ExprValue =
```

Comment:
Since this is a public function, we'll also need to deprecate this and add the alternative.